include/Zycore/Defines.h

```diff
@@ -264,8 +332,8 @@
 #   define ZYAN_NO_SANITIZE(what)
 #endif
 
-#if defined(ZYAN_MSVC) || defined(ZYAN_BORLAND)
-#   define ZYAN_INLINE __inline
+#if defined(ZYAN_BORLAND)
+#   define ZYAN_INLINE static __inline
```

Comment:
Old version resulted in storage class inconsistency between different compilers.